src/api/auth/authApiInternal.ts

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext, IParsedError, maskUserInfo, parseError } from '@microsoft/vscode-azext-utils';
+import { l10n } from 'vscode';
+import { AzExtCredentialManager } from '../../../api/src/auth/credentialManager/AzExtCredentialManager';
+import { apiUtils } from '../../../api/src/utils/apiUtils';
+import { azureExtensions } from '../../azureExtensions';
+import { ext } from '../../extensionVariables';
+import { localize } from '../../utils/localize';
+
+const allowedExtensionIds: string[] = Array.from(
```

Comment:
Would it be preferable to set these extension ids separately / manually instead of building them off of the `azureExtensions` list?